src/java/org/apache/cassandra/tcm/sequences/BootstrapAndJoin.java

```diff
@@ -106,12 +109,14 @@ public static boolean bootstrap(final Collection<Token> tokens,
         }
 
         Future<StreamState> bootstrapStream = StorageService.instance.startBootstrap(tokens, metadata, replacingEndpoint);
+        Future<Void> accordReady = AccordService.instance().epochReady(metadata.epoch);
+        Future<?> ready = FutureCombiner.allOf(Lists.newArrayList(bootstrapStream, accordReady));
```

Comment:
this is kinda a common pattern now (at least happens more than once)... makes me wonder if we should handle this differently...  I don't actually have an idea atm, just pointing out duplication of logic.